README.md

```diff
@@ -1,70 +1,183 @@
-# Getting Started with Create React App
+<a name="Hello-React-Front-End"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+REQUIRED SECTIONS:
+- Table of Contents
```

Comment:
- [ ] Kindly make sure to add the link to the backend repo as required.

Reply:
Thank you, Updateed

Reply:
Hi @karam084, You haven't added a link to the backend n your frontend repo?